src/lib/db/migrations/20230310111029_initial.ts

```diff
@@ -63,11 +70,8 @@ export async function up(knex: Knex): Promise<void> {
     def.datetime('updated_at').notNullable().defaultTo(now())
     def.integer('token_id')
     def.primary(['id'])
-    def.specificType('hash', 'CHAR(64)').notNullable()
+    def.specificType('hash', 'CHAR(66)').notNullable()
```

Comment:
why has this increased?

Reply:
This should not longer be relevant, but wonder why do we strip/restore? if we could just store?